google/cloud/spanner/admin/integration_tests/database_admin_integration_test.cc

```diff
@@ -130,6 +130,17 @@ TEST_F(DatabaseAdminClientTest, DatabaseBasicCRUD) {
   EXPECT_THAT(database->name(), EndsWith(database_.database_id()));
   EXPECT_FALSE(database->has_encryption_config());
   EXPECT_THAT(database->encryption_info(), IsEmpty());
+
+  // Test mixin method
```

Comment:
can we move this to its own `TEST_F(...)`?

similar to https://github.com/googleapis/google-cloud-cpp/blob/291074bca1fb36966696db56f1d20f16911fe327/google/cloud/spanner/admin/integration_tests/database_admin_integration_test.cc#L657

Reply:
Done.